### PR TITLE
Update for Laravel Framework 6

### PR DESCRIPTION
**Pull request by ciungulete**

### CODE REVIEW

**Comment by sandervanhooft** [comment]
Thanks @ciungulete .

Please leave the Laravel helpers intact (i.e. `starts_with` and `ends_with`) in order to not break backwards compatibility. #52 is addressing this issue.

File: composer.json [hunk 1]
```diff
@@ -34,16 +34,16 @@
         "ext-json": "*",
         "ext-intl": "*",
         "dompdf/dompdf": "^0.8.0",
-        "illuminate/database": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-        "mollie/laravel-mollie": "^2.5",
+        "illuminate/database": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "mollie/laravel-mollie": "^2.7",
```
Comment (sandervanhooft) [inline composer.json]: No need to bump this.

Comment (sandervanhooft) [reply composer.json]: laravel-mollie ^2.5 is fine.

File: composer.json [hunk 2]
```diff
@@ -34,16 +34,16 @@
...
         "moneyphp/money": "^3.2",
         "nesbot/carbon": "^1.33 || ^2.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "~3.0",
-        "phpunit/phpunit": "^6.5|^7.5",
+        "orchestra/testbench": "~3.0|~4.0",
+        "phpunit/phpunit": "^6.5|^7.5|^8.3.4",
```
Comment (sandervanhooft) [inline composer.json]: Any reason to use `^8.3.4` instead of just `^8.3`?

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 <!--
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/mollie/laravel-cashier.svg?style=flat-square)](https://packagist.org/packages/mollie/laravel-cashier)
 -->
-[![Build Status](https://travis-ci.org/laravel/cashier-mollie.svg?branch=master)](https://travis-ci.org/laravel/cashier-mollie)
+[![Build Status](https://travis-ci.org/ciungulete/cashier-mollie.svg?branch=develop)](https://travis-ci.org/ciungulete/cashier-mollie.svg?branch=develop)
```
Comment (sandervanhooft) [inline README.md]: Please revert this

File: tests/Order/OrderNumberGeneratorTest.php [hunk 4]
```diff
@@ -37,10 +38,10 @@ public function usesConfiguredOffsetAndModelCount()
...
         $this->generator = new OrderNumberGenerator;
 
-        $this->assertTrue(ends_with($this->generator->generate(), '16'));
+        $this->assertTrue(Str::endsWith($this->generator->generate(), '16'));
 
         factory(Order::class, 3)->create();
-        $this->assertTrue(ends_with($this->generator->generate(), '19'));
+        $this->assertTrue(Str::endsWith($this->generator->generate(), '19'));
 
     }
```
Comment (sandervanhooft) [inline tests/Order/OrderNumberGeneratorTest.php]: Please revert to the original string helpers (because of backwards compatibility). #52 is solving this.

**Comment by ciungulete** [comment]
Done :-)

File: composer.json (hunk 2 above)
Comment (ciungulete) [reply composer.json]: No

File: README.md (hunk 3 above)
Comment (ciungulete) [reply README.md]: I do it

**Comment by sandervanhooft** [comment]
Thanks @ciungulete ! LGTM.

Will fix #52 first before merging this.

**Comment by ciungulete** [comment]
I try to add "laravel/helpers" in cashier-mollie composer, but it not compatible with illuminate/support 5.5, work only with versions great that 5.8... and cashier-mollie works with versions larger than 5.5. I think this patch is good.

**Comment by sandervanhooft** [comment]
Great, thanks @ciungulete !

**Pull request merged**